week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,9 +45,21 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {
-  
-}
+function formatPercentage(arr){
+function mystr(strarr) {
+    if(strarr > 100){
+        return "100%";
+    }
+    if(Number.isInteger(strarr)){
```

Comment:
This is really close to being right but will only work for Integers (e.g. 1,2,3,4) but not for Floats (e.g 1.1, 31.21, 3133.131).

Do you know how you'd fix this?